src/fragments/lib/auth/common/device_features/common.mdx

```diff
@@ -15,7 +15,7 @@ amplify auth console
 **Select the following option** to open the Cognito User Pool console:
 ```bash
 ? Which Console
-    User Pool
+  User Pool
```

Comment:
Does this count as code indentation? I'm not sure what the updated guidelines are around CLI prompts

Reply:
CLI prompts (living in a terminal) are not the same as code blocks (which live in a code editor). Reflect the terminal. If things are not indented, don't indent. If they are, use the same spacing. 

Can you provide a screenshot of how this dialog looks in the terminal?